netbox_secrets/api/views.py

```diff
@@ -202,7 +202,7 @@ def create(self, request):
         if master_key is None:
             return HttpResponseBadRequest(ERR_PRIVKEY_INVALID)
 
-        current_session_key = self.queryset.first()
+        current_session_key = self.get_queryset().first()
```

Comment:
This should not be bypassed, instead we should look into why `self.queryset` does not contain the correct `QuerySet` defined in `get_queryset`

Reply:
Could it be due to: https://github.com/Onemind-Services-LLC/netbox-secrets/blob/dcedeba01920702c856e919065b23d3f1d0a9611/netbox_secrets/models/secrets.py#LL177C9-L177C18 ?